.travis/testBuildLicenseList.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+TOOLSJAR=spdx-tools-2.1.8-jar-with-dependencies.jar
+LICENSEXMLDIR=./
+TESTDIR=./test/original
+LICENSEOUTPUTDIR=.OUTPUTFILES
+VERSION=$(git describe)
+if [[ $? != 0 ]]; then VERSION="UNKNOWN"; fi
+RELEASEDATE=$(date -I)
+mkdir $LICENSEOUTPUTDIR
+# Create a file to skip expected warnings
+IGNOREWARNINGS="\"Duplicates licenses: AGPL-3.0-or-later, AGPL-3.0-only\",\"Duplicates licenses: GFDL-1.1-or-later, GFDL-1.1-only\",\"Duplicates licenses: GFDL-1.2-or-later, GFDL-1.2-only\",\"Duplicates licenses: GFDL-1.3-or-later, GFDL-1.3-only\",\"Duplicates licenses: GPL-1.0-or-later, GPL-1.0-only\",\"Duplicates licenses: GPL-2.0-or-later, GPL-2.0-only\",\"Duplicates licenses: GPL-3.0-or-later, GPL-3.0-only\",\"Duplicates licenses: LGPL-2.0-or-later, LGPL-2.0-only\",\"Duplicates licenses: LGPL-2.1-or-later, LGPL-2.1-only\",\"Duplicates licenses: LGPL-3.0-or-later, LGPL-3.0-only\",\"Duplicates licenses: MPL-2.0, MPL-2.0-no-copyleft-exception\""
+WARNINGSFILE="TEMP-WARNINGS"
+echo $IGNOREWARNINGS > $WARNINGSFILE
+# Test and generate the output files
+PUBLISHCMD="LicenseRDFAGenerator $LICENSEXMLDIR $LICENSEOUTPUTDIR $VERSION $RELEASEDATE $TESTDIR $WARNINGSFILE"
```

Comment:
Generating RDFa seems unnecessary for *validation*.  Would it be hard to teach spdx/tools a mode that just validates, without writing output?  On a related note, it's not clear to me whether successful RDFa generation is sufficient to validate the XML.  Is it possible that RDFa generation could succeed, but JSON (or whatever) generation would fail?

Reply:
> Generating RDFa seems unnecessary for validation. Would it be hard to teach spdx/tools a mode that just validates, without writing output? 

Yes - more work than I would prefer to do at this point, but not impossible.

If we want to dramatically improve performance, we should only test and translate the XML files which have been added or modified.  Currently, it rebuilds all files.  This, however, is a lot of work to implement.